src/opdsrequestmanager.cpp

```diff
@@ -49,19 +48,62 @@ void OpdsRequestManager::doUpdate(const QString& currentLanguage, const QString&
     qInfo() << "Downloading" << url.toString(QUrl::FullyEncoded);
 
     QNetworkRequest request(url);
-    if (mp_reply) {
-        mp_reply->abort();
-    }
-    mp_reply = m_networkManager.get(request);
-    connect(mp_reply, &QNetworkReply::finished, this, &OpdsRequestManager::receiveContent);
+    auto mp_reply = m_networkManager.get(request);
```

Comment:
Could we set the builded url (before this line) with `opdsResponseFromPath` ?

Reply:
Im divided on whats the better way here
1. Have 2 functions:
`opdsResponseFromPath(path)` (which calls the next one with an empty query)
`opdsResponseFromPath(path, query)`

2. Have a default empty query parameter
`opdsResponseFromPath(path, query = "")`